src/app/GitCommands/Settings/AppSettings.cs

```diff
@@ -2087,11 +2089,9 @@ public static bool GitAsyncWhenMinimized
             get => GetBool("GitAsyncWhenMinimized", false);
         }
 
-        public static bool IsEditorSettingsMigrated
-        {
-            get => GetBool("Git.IsEditorSettingsMigrated", false);
-            set => SetBool("Git.IsEditorSettingsMigrated", value);
-        }
+        private static readonly SettingsPath HiddenSettingsPath = new AppSettingsPath("Hidden");
+        private static readonly SettingsPath MigrationSettingsPath = new AppSettingsPath(HiddenSettingsPath, "Migration");
+        public static ISetting<bool> IsEditorSettingsMigrated { get; set; } = Setting.Create(MigrationSettingsPath, nameof(IsEditorSettingsMigrated), false);
```

Comment:
It is an helper instance with the actual property (refer to other usages).
```suggestion
        public static ISetting<bool> IsEditorSettingsMigrated { get; } = Setting.Create(MigrationSettingsPath, nameof(IsEditorSettingsMigrated), false);
```